src/datetime/mod.rs

```diff
@@ -516,12 +623,16 @@ impl<Tz: TimeZone> DateTime<Tz> {
         result
     }
 
-    /// Return an RFC 3339 and ISO 8601 date and time string with subseconds
-    /// formatted as per `SecondsFormat`.
+    /// Return an RFC 3339 and ISO 8601 date and time string with subseconds formatted as per
+    /// `SecondsFormat`.
     ///
     /// If `use_z` is true and the timezone is UTC (offset 0), uses `Z` as
     /// per [`Fixed::TimezoneOffsetColonZ`]. If `use_z` is false, uses
-    /// [`Fixed::TimezoneOffsetColon`]
+    /// [`Fixed::TimezoneOffsetColon`].
+    ///
+    /// Note that if the year of the `DateTime` is outside of the range 0 through 9999 the date
+    /// while be formatted as an expanded representation according to ISO 8601. This makes the
```

Comment:
grammar. I suggest:

    /// Note that if the year of the `DateTime` is outside of the range 0 through 9999 then
    /// the date will be formatted as an expanded representation according to ISO 8601.